source/lib/storage.js

```diff
@@ -74,3 +75,13 @@ export function setCredentialsValid(valid: boolean) {
 export function getCredentialsValid(): Promise<boolean> {
   return getItemAsBoolean(credentialsValidKey)
 }
+
+/// MARK: Favorite Buildings
+
+const favoriteBuildingsKey = 'buildings:favorited'
+export function setFavoriteBuildings(buildings: Array<BuildingType>) {
+  return setItem(favoriteBuildingsKey, buildings)
+}
+export function getFavoriteBuildings() {
```

Comment:
I'd like a return type on this function, please; probably `Promise<Array<BuildingType>>` given this code. It should return the same type as what was stored, essentially.